src/types/metadata.ts

```diff
@@ -75,12 +75,13 @@ export type TypeId = keyof Metadata;
  */
 export interface TokenMetadata {
   address: Address;
-  categories?: string[];
-  description?: string;
-  website?: string;
+  categories?: string[] | null;
+  description: string;
+  website: string;
```

Comment:
sure this data always exist?

Reply:
Double checked and https://meta.yearn.network/tokens/1/all never returns `undefined` or `null` for those keys

```typescript
const json = [{}]; // the whole return from the endpoint above

json.find((j) => j.website === undefined);  // undefined

json.find((j) => j.description === undefined); // undefined
```

Happy to make it optional if you have concerns that might change in the future

Reply:
Since there might be situations where a vault is endorsed in the blockchain, but descriptions have not yet been filled out, I have my doubts with the change since Im not sure. Current state does not mean is the state always, and that is why I mostly trust the person who wrote the initial code.

Lets wait for @jstashh opinion

Reply:
Don't think it's explicitly required in meta - https://github.com/yearn/yearn-meta/blob/master/schema/token.json#L123

But I would assume that it'd be present, doesn't mean someone could forget to add it though